src/types/plugin-input.ts

```diff
@@ -27,37 +26,25 @@ export const startStopSchema = T.Object({
   miscellaneous: T.Object(
     {
       startRequiresWallet: T.Boolean(),
-      maxConcurrentTasks: T.Array(
+      maxConcurrentTasks: T.Record(
```

Comment:
If using a Record here the string is our `userRole` and it's value could be the limit instead of `{limit: 0}`. Finding the limit could also be direct access instead of using `find()`

Reply:
Makes sense done. Thanks